frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -296,3 +299,16 @@ export const getTaskStatus = (pipelinerun: PipelineRun, pipeline: Pipeline): Tas
   }
   return taskStatus;
 };
+
+export const getResourceModelFromTask = (task: PipelineTask): K8sKind => {
+  const {
+    taskRef: { kind },
+  } = task;
+
+  switch (kind) {
+    case ClusterTaskModel.kind:
+      return ClusterTaskModel;
+    default:
+      return TaskModel;
+  }
```

Comment:
If its just going to be these two models, instead of using switch can we do this
```
return kind ===  ClusterTaskModel.kind ?  ClusterTaskModel : TaskModel
```
and reduce it to just one line?

Reply:
Fair. I viewed this as an extendable model, but I can't really say there will likely be any other types of tasks at this time.